Packages/io.chainsafe.web3-unity/Runtime/Scripts/Model/AllNftModel.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ChainSafe.Gaming.UnityPackage.Model
+{
+    public class AllNftModel
```

Comment:
Was this necessary to be a class?

Reply:
yes, allowing it do be deserialized into a class gives the users heaps of freedom when navigating the json responses. It's how we've always done it with the API calls, if you check the models in the runtime folder you'll see. If you'd like it done another way just let me know <3

Reply:
Doesn't really matter anyway as it now needs to be redone with the API endpoint being decommissioned this morning :|